client/webserver/site/src/js/markets.ts

```diff
@@ -872,10 +889,6 @@ export default class MarketsPage extends BasePage {
     // depth chart
     ws.request('loadmarket', makeMarket(host, base, quote))
 
-    // candlesticks
-    this.candleDur = fiveMinBinKey
-    this.loadCandles()
```

Comment:
Moved this to `setCandleDurBttns` since we were always undoing our work there so I think it's better it stays there. Resolves https://github.com/decred/dcrdex/pull/2284/files#r1174081327